Libraries/Components/Keyboard/Keyboard.js

```diff
@@ -155,12 +158,12 @@ Keyboard = KeyboardEventEmitter;
 Keyboard.dismiss = dismissKeyboard;
 Keyboard.scheduleLayoutAnimation = function(event: KeyboardEvent) {
   const {duration, easing} = event;
-  if (duration) {
+  if (duration != null) {
```

Comment:
This changes the behavior in the case that duration is 0. If you add the explicit check for that then flow should be okay.

```
if (duration != null && duration != 0) {
```